CHANGELOG.md

```diff
@@ -1,19 +1,46 @@
+## 1.8.0-beta.1
+
+### ✨ Features and improvements
+* Level-of-detail support for tile coverage ([#8975](https://github.com/mapbox/mapbox-gl-js/pull/8975))
```

Comment:
While the PR was merged, it states that:
> The maximum pitch angle is left unchanged so this PR has no immediate impact on behavior. coveringTiles should provide similiar results as the original implementation when pitch is < 60

Given then that the PR is an internal improvement only, does it warrant a changelog entry? If we think it does, might I suggest:

```suggestion
* Implemented a new tile coverage algorithm to enable level-of-detail support in a future release ([#8975](https://github.com/mapbox/mapbox-gl-js/pull/8975))
```